### ISSUE TITLE
Remove unnecessary @exceptions for required parameters

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium#3327]
The exception is implicit in the `@param`.

For example:
```
* @exception {DeveloperError} vector must be defined.
```
in AttributeCompression.js can be removed.

I suspect this is only in a few places.

**Comment by mramato** [comment]
We actually already did this once #1449, so I doubt there are many more than this (if any).

**Comment by AmayAgrawal** [comment]
Sir,is this issue already solved or not?If not Can u please explain in little brief where I have to make such changes

**Comment by pjcozzi** [comment]
It's not solved yet.

To fix this issue, we want to remove doc for exceptions like this:
```
@exception {DeveloperError} vector must be defined.
```
when a function's parameter is required, e.g.,
```
@param {Cartesian3} vector The normalized vector to be compressed into 2 byte 'oct' encoding.
```
Most likely only [AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js) needs to be changed.

For an example of when we did this same fix before, see #1449.

**Comment by AmayAgrawal** [comment]
@param {Object} options Object with the following properties:
     * @param {Cartesian3} options.dimensions The width, depth, and height of the box stored in the x, y, and z coordinates of the <code>Cartesian3</code>, respectively.
 * @exception {DeveloperError} All dimensions components must be greater than or equal to zero.

Sir what about this type of files

**Comment by pjcozzi** [comment]
You are right, only the "must be defined" exceptions, not the "must be normalized" or other exceptions.  Those are useful documentation.  The "must be defined" exceptions are redundant with the parameter documentation.

**Comment by pjcozzi** [comment]
Fixed in #3409.
